### PR TITLE
added inline react component support for svg files (#397) (#206)

### PR DESCRIPTION
**Pull request by jbhoosreddy**
I worked on setting up the inline svg images. @jasonLaster let me know your thoughts on this approach.

### CODE REVIEW

**Comment by jlongster** [comment]
Feel free to ping me for reviews too. I'd like to review this as I've researched this approach a while back and found some flaws with it, but I can't remember. I'll look at it on Monday.

**Comment by jasonLaster** [comment]
Thanks @jbhoosreddy for doing the leg work.

@jlongster this looks good to me as long as it works well for the firefox panel.

**Comment by jbhoosreddy** [comment]
if this is fine, I'll look into which other icons I can migrate from Isvg to this approach, and add that to this PR.

**Comment by jasonLaster** [comment]
@jbhoosreddy thanks, i don't see how that can hurt. That way we can evaluate it holistically.

File: public/images/Icons.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const React = require("react");
+const InlineSVG = require("svg-inline-react");
+const { DOM: dom } = React;
+
+const DomainIcon = props => {
```
Comment (jlongster) [inline public/images/Icons.js]: Any reason we need all these components now? We could just use `InlineSVG({ src: DomainIcon })` directly in the component that needs it. That would reduce the need to manage this file.

**Comment by jlongster** [comment]
I'm a little concerned that [svg-inline-loader](https://github.com/sairion/svg-inline-loader) is deprecated. But this looks pretty good. It's somewhat similar to what I was thinking of doing. I was going to write a webpack loader that loads SVG and transforms it directly into a React component. 

I'm fine using this for now though! If we run into problems we could fork the library and maintain it.

File: public/images/Icons.js (hunk 1 above)
Comment (jasonLaster) [inline public/images/Icons.js]: This seems like a nice improvement in another PR.

**Comment by jasonLaster** [comment]
> I'm a little concerned that svg-inline-loader is deprecated

I am too, but had a similar reaction. I think it would be easy enough to maintain a library for our own purposes.

**Pull request merged**